tests/auth_tests/client.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+from django.contrib.auth.views import INTERNAL_RESET_TOKEN
+from django.test import Client
+
+
+def extract_token_from_url(url):
+    token_search = re.search(r'/reset/.*/(.+?)/', url)
+    if token_search:
+        return token_search.group(1)
+
+
+class PasswordResetConfirmClient(Client):
```

Comment:
`test_confirm_link_redirects_to_secure_confirm_page` was created for that purpose. Do you want to explicitly test the `PasswordResetConfirmClient` client?